File: LatPrak2_2 (PHP)/Hardware.php
 <?php 
 
 class Hardware extends Product{

	//atribut dalam class Hardware
	private $brand = "";
	private $model = "";

		
	//constructor
	public function __construct($brand = "", $model = ""){
		$this->price = "";
		$this->id_product = "";
	}
		
	//set and get for atribut brand
	public function setBrand($brand){
		$this->brand = $brand;
	}

	public function getBrand(){
		return $this->brand;
	}

	//set and get for atribut model
	public function setModel($model){
		$this->model = $model;
	}

	public function getModel(){
		return $this->model;
	}

	function __destruct(){
	}

}

?>

File: LatPrak2_2 (PY)/Hardware.py
from Product import Product

class Hardware(Product):
#kelas yang digunakan untuk mengimplementasikan kelas Hardware

	#atribut
	__brand = "";
	__model = "";

	#constructor
	def __init__(self, brand = "", model = ""):
		self.brand = brand
		self.model = model
		

	#get dan set untuk atribut price
	def setBrand(self, brand):
		self.brand = brand

	def getBrand(self):
		return self.brand
	
	#get dan set untuk atribut Model
	def setModel(self, model):
		self.model = model

	def getModel(self):
		return self.model
	
	
	